snapcraft/meta.py

```diff
@@ -123,10 +122,11 @@ def _compose_snap_yaml(meta_dir, config_data):
             snap_yaml[key_name] = config_data[key_name]
 
     if 'apps' in config_data:
-        apps = config_data['apps']
-        apps = _wrap_apps(apps)
-        snap_yaml['apps'] = \
-            _copy_security_profiles(meta_dir, apps)
+        snap_yaml['apps'] = _wrap_apps(config_data['apps'])
+
+    if 'uses' in config_data:
+        snap_yaml['uses'] = \
+            _copy_security_profiles(meta_dir, config_data['uses'])
```

Comment:
Should you also copy over the `offers` here since, as you say, you don't want to block them from being used even if we don't have a good definition for them?